src/__test__/pet.test.js

```diff
@@ -0,0 +1,183 @@
+const Pet = require('../../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+  });
+
+  describe('constructor', () => {
+    it('sets the name property', () => {
+      const pet = new Pet('Fido');
+  
+      expect(pet.name).toEqual('Fido');
+    });
+  });
+
+  describe('constructor', () => {
+  
+    it('has a initial age of 0', () => {
+      const pet = new Pet('Fido');
+  
+      expect(pet.age).toEqual(0);
+    });
+  });
+
+  describe('growUp', () => {
```

Comment:
Theres quite a lot of indentation to be fixed here, as above not going to list each instance here but you should definitely run prettier across this